hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadata.java

```diff
@@ -138,12 +138,15 @@ static HoodieBackedTableMetadata createHoodieBackedTableMetadata(HoodieEngineCon
   FileStatus[] getAllFilesInPartition(Path partitionPath) throws IOException;
 
   /**
-   * Fetch list of all partitions path that whose relative partition paths match the given prefixes
+   * Fetch list of all partitions path that whose relative partition paths is under the
```

Comment:
nit: `Fetch list of all partitions paths that are the sub-directories of the list of provided (relative) paths`

Reply:
Fixed.